scripts/functions.py

```diff
@@ -1,27 +1,74 @@
 #####################################
-## This file is for functions common between multiple files
+## This file is for functions and constants common between multiple files
 #####################################
 
 import datetime
 import pandas as pd
 from collections import defaultdict
+import yaml
+
+#####################################
+# NON-CONFIGURABLE CONSTANTS
+#####################################
+
+# Province to Time Zone Mapping
+PROVINCIAL_TIME_ZONES = {
+    'BC':0,
+    'AB':1,
+    'SAS':2,
+    'MAN':2,
+    'ONT':3,
+    'QC':3,
+    'NB':4,
+    'NL':4,
+    'NS':4,
+    'PEI':4}
+
+# Residual Hydro Capacity (GW) per province in 2017

```

Comment:
> Did you have a reason for moving these here?

Yes, they are also used in CapacityFactor.py. You did ask if we should maybe merge the two files, but I argued against.

> Or maybe since this is called a functions.py file global constants shouldnt be in here?

Good point, this is definitely arguable. Putting the constants in the yaml was a no-go in your opinion, because they aren't constants that should ever be configured (which is fair). Putting them in a new constants.py or constants.yaml file was another option. Leaving them as-is was an option, but that means leaving the copy-pasted code (which is worse practice than adding global constants). On balance, I felt adding them to functions.py would be less overkill than making a new file.

Reply:
You sure they are actually used in CapacityFactor.py tho? There is a bigger issue on how to handle availability factor with Hydro though (see issue #25), so we can leave it as you suggested for now. 

Thanks for the explanation! I like your idea of a constants.py or .yaml (as long as its separate from the file the user interacts with)